pygmt/src/sph2grd.py

```diff
@@ -0,0 +1,68 @@
+"""
+sph2grd - Compute grid from spherical harmonic coefficients
+"""
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    I="increment",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(I="sequence", R="sequence")
+def sph2grd(table, **kwargs):
```

Comment:
Not sure when https://github.com/GenericMappingTools/pygmt/issues/1479 will be decided, but please make sure the input parameter is consistent and include in the docstring.

Reply:
Updated in 974f908

Reply:
Time to change it to 'data' as per #1479.
```suggestion
def sph2grd(data, **kwargs):
```